ci/workflows/main-cron.yml

```diff
@@ -236,7 +236,7 @@ steps:
 
   - label: "unit test (madsim)"
     key: "unit-test-deterministic"
-    command: "MADSIM_TEST_NUM=100 timeout 15m ci/scripts/deterministic-unit-test.sh"
```

Comment:
I prefer to keep this `timeout` command first, until we actually set a timeout policy.
Because buildkite's timeout policy is not strict, it can fluctuate by 1minute. This timeout is strict in contrast,.
So sometimes people may just retry it, instead of bumpiing the timeout, and subsequently other folks get blocked and not sure if it's their PR which caused it.